test/builder-api/src/channels.js

```diff
@@ -119,6 +119,34 @@ describe('Channels API', function () {
         });
     });
 
+
+    it('returns latest packages in a channel fails without a target', function (done) {
+      request.get('/depot/channels/neurosis/foo/pkgs/latest')
+        .type('application/json')
+        .accept('application/json')
+        .expect(400)
+        .end(function (err, res) {
+          expect(res.text).to.be.empty;
+          done(err);
+        });
+    });
+
+    it('returns latest packages in a channel', function (done) {
+      request.get('/depot/channels/neurosis/foo/pkgs/latest?target=x86_64-linux')
+        .type('application/json')
+        .accept('application/json')
+        .expect(200)
+        .end(function (err, res) {
+          expect(res.body.channel).to.equal('foo');
+          expect(res.body.target).to.equal('x86_64-linux');
+//          expect(res.body.data).length.to.equal(1);
```

Comment:
Given the above failure case, it may be worth adding a second version of the 'testapp' package, as well as possibly a differently named package to make assertions that our `distinct_on` is working correctly.